database/things/oz.pldb

```diff
@@ -151,7 +151,9 @@ rijuRepl https://riju.codes/oz
       {System.showInfo 'Hello, world!'}
       {Application.exit 0}
   end
-  
+
+indeedJobs oz engineer
+ 2022 19
```

Comment:
The latest stable release of Oz was in 2018 (https://en.wikipedia.org/wiki/Oz_%28programming_language%29). Might this number be too high?